instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/server/SpringWebfluxTest.java

```diff
@@ -142,7 +144,13 @@ void basicGetTest(Parameter parameter) {
                               stringKey("spring-webflux.handler.type"),
```

Comment:
is `spring-webflux.handler.type` redundant now?

Reply:
yeah i think it is redundant now. but not sure what is the recommended procedure of deprecating a metric? can we just remove it or we should mark it deprecated and remove it in a future release? thanks!

Reply:
since it's already an opt-in experimental attribute, I think it's ok to just remove it and mention in the PR title so that I will remember to add it in the migration notes when writing the change log

Reply:
i have removed it in the latest change, and updated the title. please kindly have a look. thanks!